src/schemas/v2.ts

```diff
@@ -4,6 +4,7 @@ export type Intervention = Route | Crossing;
 
 export interface Crossing {
   name?: string;
+  description?: string;
```

Comment:
Reminder this is autogenerated. I edited the JSON, then `npm run generate-form-ts`